roles/configure-kafka/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
plz kindly update README , this content is autogenerated ;)

Reply:
Working on it

Reply:
Updated readme